version/version_test.go

```diff
@@ -0,0 +1,63 @@
+package version
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestGetVersion(t *testing.T) {
+	vi := GetVersion()
+	assert.Equal(t, Version, vi.Version)
+	assert.Equal(t, Prerelease, vi.Prerelease)
+}
+
+func TestVersionInfo_VersionNumber(t *testing.T) {
```

Comment:
Tests for this seemed like they might be a little over the top, but I thought it would be worth some basic verification that we're producing a semantic version. In other words, `"<version core>[-<pre-release>][+<metadata]"`.